### DISCUSSION TITLE
Can not scroll when too many item in navigation menu

### DISCUSSION THREAD

**Discussion by tungthanhss** [ng-alain/ng-alain#2465] [General]
When the menu is too much child item. I can not scroll to select them. Anyone help me ?
Thank you
![image](https://github.com/ng-alain/ng-alain/assets/34281514/987d796f-d251-486a-b5b9-eb71b8cc00bf)

**Comment by cipchk** [comment]
Can you provider a `@delon/*` version. or try upgrade to latest.

**Reply by tungthanhss** [reply]
i'm using version 16.4.2

**Reply by cipchk** [reply]
Similar problems have been fixed before, the relevant situation is in [#1692](https://github.com/ng-alain/delon/pull/1692/files).

Try adding this to `styles.less` in your NG-ALAIN project:

```less
.alain-default__aside-inner {
    overflow: hidden scroll;
}
```
